### PR TITLE
[TACHYON-1443] Tachyon does not work well with mesos framework authentication

### PR DESCRIPTION
**Pull request by BrickXu**
Support framework authentication and role.

This PR addresses: https://tachyon.atlassian.net/browse/TACHYON-1443

### CODE REVIEW

**Comment by BrickXu** [comment]
upgrade docs.

File: common/src/main/java/tachyon/Constants.java [hunk 1]
```diff
@@ -158,6 +158,14 @@
       "tachyon.integration.worker.resource.cpu";
   public static final String INTEGRATION_WORKER_RESOURCE_MEM =
       "tachyon.integration.worker.resource.mem";
+  public static final String INTEGRATION_MESOS_ROLE =
```
Comment (jsimsa) [inline common/src/main/java/tachyon/Constants.java]: keep the definition in this block sorted alphabetically

File: docs/Configuration-Settings.md [hunk 2]
```diff
@@ -712,6 +712,26 @@ configuration options.
   <td>Memory resource required to run a Tachyon worker. This memory does not include the memory
   configured for tiered storage.</td>
 </tr>
+<tr>
```
Comment (jsimsa) [inline docs/Configuration-Settings.md]: keep the table entries sorted alphabetically

File: docs/Configuration-Settings.md [hunk 3]
```diff
@@ -712,6 +712,26 @@ configuration options.
...
+  <td>tachyon.integration.mesos.role</td>
+  <td>*</td>
```
Comment (jsimsa) [inline docs/Configuration-Settings.md]: set the default value for this and the other properties in the /common/src/main/resources/tachyon-default.properties file

Comment (jsimsa) [inline docs/Configuration-Settings.md]: again, keep that file sorted alphabetically (within each block)

**Comment by jsimsa** [comment]
ok to test

**Comment by jsimsa** [comment]
@BrickXu thanks for the improvement! ... left some minor comments, in addition there are two other things:

1) your commits are not linked to your github account, fix that by following these instructions: https://help.github.com/articles/changing-author-info/

2) provide a link to the JIRA ticket in the PR description

**Comment by BrickXu** [comment]
@jsimsa thanks for reviewing the PR. I will upgrade this comments later.

**Comment by BrickXu** [comment]
@jsimsa done.

**Comment by jsimsa** [comment]
LGTM

**Comment by jsimsa** [comment]
@BrickXu thanks for the PR

**Pull request merged**